src/containers/documentation/swaggerPlugins/CurlForm.scss

```
@@ -1,23 +1,7 @@
-.curl-container {
-  margin: 0 2em 2em;
-  padding-top: 1em;
-  #title {
-    margin-top: 0 !important;
-    margin-bottom: 0 !important;
-  }
-  .curl-form {
-    background-color: #EEEEEE;
-    border-radius: 6px;
-    border-style: solid;
+.va-api-curl-form {
+  border-radius: 6px;
+}
 
-    &__contents {
-      margin: 1em;
-      .copy-to-clipboard {
-        cursor: pointer;
-      }
-    }
-    .highlight-code {
-      display: flex;
-    }
-  }
+.va-api-copy-to-clipboard {
```

I think this is a good place to follow BEM. This class seems like something we could conceivably use in other places on the site, but we probably will want different styles for the curl form (since it's inside of swagger-ui).
```suggestion
.va-api-curl-form__copy-to-clipboard {
```